azure-pipelines.yml

```diff
@@ -28,7 +28,7 @@ steps:
   - script: python -m pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/cu117
     displayName: 'Install PyTorch'
 
-  - script: python -m nebullvm.installers.auto_installer --backends all --compilers all
+  - script: python -m nebullvm.installers.auto_installer --frameworks all --extra-backends all --compilers all
```

Comment:
Can we test specifying just the compilers keyword (as  it is done in the quick-installation guide)?

```bash
python -m nebullvm.installers.auto_installer --compilers all
```